processor/processor.go

```diff
@@ -1581,6 +1604,27 @@ func (proc *Handle) processJobsForDest(partition string, subJobs subJob) *transf
 		proc.logger.Debug("[Processor] Total jobs written to event_schema: ", len(eventSchemaJobs))
 	}
 
+	if len(archivalJobs) > 0 {
```

Comment:
Storing `eventSchemaJobs` and `archivalJobs` could happen in parallel using an `errgroup.Group`. Then we could  panic once, in case `g.Wait()` returns an error

Reply:
Addressed this too